tsconfig.json

```diff
@@ -16,11 +16,7 @@
     "declaration": true,
     "emitDeclarationOnly": true,
     "composite": true,
-    "incremental": true,
-    "paths": {
-      "waku": ["./packages/waku/src/main.ts"],
-      "waku/*": ["./packages/waku/src/*.ts"]
-    }
```

Comment:
I found that examples are aliases to the src file, I think this is not good. we should type check from output